src/sensor/ping360.h

```diff
@@ -236,6 +236,92 @@ class Ping360 : public PingSensor
     }
     Q_PROPERTY(int speed_of_sound READ speed_of_sound WRITE set_speed_of_sound NOTIFY speedOfSoundChanged)
 
+    /**
+     * @brief Returns the angle offset from sample position
+     *
+     * @return int
+     */
+    int angle_offset() { return _angle_offset; }
```

Comment:
Consider if this should be a property of the visualizer, so we can do a transform of all the existing data inside the visualizer buffer.

Reply:
Added TODO.